config/locales/en.yml

```diff
@@ -852,6 +852,9 @@ en:
       is_active: "Is Active?"
       contact: "The following contact information will appear on statements and should indicate where payment should be sent"
       show_instrument_availability: Show Instrument Availability Status?
+      order_notification: |
+        The email address that will receive order notifications
```

Comment:
I think this text could be a little clearer. Without knowing, it's unclear what a "order notification" would be. "This email address will be notified whenever an order is placed by a user" or something similar. The next line is fine.